assets/src/edit-story/components/animationTimeline/ruler.js

```diff
@@ -34,18 +34,20 @@ const Path = styled.path`
 `;
 
 const Text = styled.text`
-  font-family: Roboto;
+  font-family: Roboto, sans-serif;
   font-size: 13px;
```

Comment:
Shouldn't font family & size be in the theme anyway?

Reply:
I can pull that in as a variable.

Reply:
The designs actually show Google Sans which I think are no longer accurate as I believe we are using Roboto throughout.